src/Backend/Modules/Blog/Engine/Model.php

```diff
@@ -319,15 +313,11 @@ public static function existsCategory(int $id): int
         );
     }
 
-    public static function existsComment(int $id): int
+    public static function existsComment(int $id): bool
     {
-        return (bool) BackendModel::getContainer()->get('database')->getVar(
-            'SELECT 1
-             FROM blog_comments AS i
-             WHERE i.id = ? AND i.language = ?
-             LIMIT 1',
-            [(int) $id, BL::getWorkingLanguage()]
-        );
+        $repository = BackendModel::get('doctrine.orm.default_entity_manager')->getRepository(Comment::class);
```

Comment:
idem

Reply:
fixed in cd9cef81ef540cf410bc31750966a00591aa06ae